modules/@apostrophecms/doc/index.js

```diff
@@ -735,6 +880,61 @@ module.exports = {
             }
           });
         });
+      },
+      addDraftPublishedMigration() {
+        self.apos.migration.add('add-draft-published', async () => {
```

Comment:
I just tried:

* Sync down a3-demo from staging
* Fire it up
* Make a subpage, save draft don't publish
* Try to publish
* Succeeded in publishing

It looks like the migration worked for me.

So since I don't have a means of reproducing this issue, I suggest we keep an eye out for a way to reproduce it and move on.

Reply:
Yeah, trying again from the current 3.0 branch I didn't have any issues. 👍